src/database.h

```diff
@@ -101,20 +101,26 @@ class Database : public QObject
 
    //! \brief Create a blank database in the given file
    static bool createBlank(QString const& filename);
-   
+
    //! backs up database to 'dir' in chosen directory
    static bool backupToDir(QString dir);
 
    //! \brief Reverts database to that of chosen file.
    static bool restoreFromFile(QString newDbFileStr);
 
+   static bool testConnection( Brewtarget::DBTypes testDb, QString const& hostname,
```

Comment:
Rename `isConnectionGood()` or something that reflects what the return value `true` means.